README.md

```diff
@@ -1,207 +1,342 @@
 # LUKSO CLI
->⚠️ DO NOT USE YET, this is WIP!
 
+> ⚠️ DO NOT USE YET, this is WIP!
```

Comment:
Please replace that with the old readme, so we can merge this. to add the release flow

Reply:
I reapplied the one from origin/main.
The changes I added...
1. doc the version command.
2. doc the additional folders.
3. doc the install script for linux/mac
4. remove install section for windows since there is no windows build/install
5. add pr preview install docs
Should I move these into a different file or area?

Reply:
Can I mark this as resolved so it can be approved and merged?